### PR TITLE
[8.0] docs: clarify how to install TokenManager in v8

### LINKED ISSUE #7274: [v8.0.31] Unexpected configuration result when installing TokenManager
Hi,

I just followed fairly blindly the instructions given here:
https://dirac.readthedocs.io/en/integration/AdministratorGuide/HowTo/pilotsWithTokens.html
My tornado config (which existed before installing this service) reads:
Tornado/Port = 8444

The above instructions tell me to install the TokenManager using 
```
install service Framework TokenManager
```
No error. This results in the service being installed the 'classic' way:
```
[dirac@dirac00 ~]$ ps aux | grep Token
dirac     3458 84.0  0.6 328652 48612 ?        Rl   14:39   0:00 /opt/dirac/diracos/bin/python3.11 /opt/dirac/diracos/bin/dirac-service Framework/TokenManager --cfg /opt/dirac/etc/Framework_TokenManager.cfg
dirac    20860  0.0  0.0   4232   448 ?        R    14:28   0:00 runsv Framework_TokenManager
```

but at the same time the configuration generated reads:
```
TokenManager = https://dirac00.grid.hep.ph.ic.ac.uk:8443/Framework/TokenManager
```
(note the port).
If this is meant to run exclusively as a tornado service, could that please be reflected in the instructions, and why does it have the incorrect port number ?

### PR DESCRIPTION
**Pull request by fstagni**
closes #7274 

BEGINRELEASENOTES

*docs
FIX: clarify how to install TokenManager in v8

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst [hunk 1]
```diff
@@ -67,18 +67,9 @@ Setting up an ``IdProvider``
 Launching the ``TokenManagerHandler``
 -------------------------------------
 
-Run the following commands from a DIRAC client to install the ``Framework/TokenManager`` Tornado service:
+Install the ``Framework/TokenManager`` Tornado service, following :ref:`these instructions <httpsTornado>`
```
Comment (andresailer) [inline docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst]: Better link here? https://dirac--7275.org.readthedocs.build/en/7275/DeveloperGuide/TornadoServices/index.html#install-a-service

Comment (fstagni) [reply docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst]: Do I need to add another label in target or is there some hidden magic that I don't know?

Comment (andresailer) [reply docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst]: You need to add a label.

cf. https://stackoverflow.com/a/5889200

I think I tried autosectionlabel, but there were too many clashes?

**Comment by fstagni** [comment]
@andresailer OK now?

**Review by andresailer — APPROVED** [review]
Thanks!

**Pull request merged**